src/css/templates/site-search.scss

```diff
@@ -6,23 +6,12 @@ body.site-search {
       display: block;
       flex-direction: row;
     }
-  }
-
-  .site-search-form {
-    margin-bottom: $spacing-xl;
-
-    input.site-search-field {
-      background-image: url('/static/img/icons/search-black.svg');
-      background-position: 3px 3px;
-      background-repeat: no-repeat;
-      background-size: 24px auto;
-      height: 2em;
-      margin-bottom: $spacing-md;
-      padding: $spacing-xs $spacing-xs $spacing-xs $spacing-xl;
-      width: 100%;
 
+    .mzp-l-sidebar {
       @media #{$mq-md} {
-        width: 80%;
+        // hidden for non-sm viewports
+        display: none;
+        visibility: hidden;
```

Comment:
Not sure we need both. Display none hides it from everyone and removes it from the DOM. Seems like the visibility property is redundent?